src/data/local/github/tracks-node/track-2/steps/step-2.md

```diff
@@ -0,0 +1,18 @@
+# Step 2 - The concepts
+
+Let's present all the concepts and tools we're going to use on your server onboard. The are a lot of them, but don't be frightned, this is why this onbaord exists. Some of them may be familiar to you depending on your background.
```

Comment:
"There are a lot of them, but don't be frightened, this is why this onboard exists."